src/utils.js

```diff
@@ -79,12 +80,19 @@ const fileNameFlag = Flags.string({
 
 const startTimeFlag = Flags.string({
 	description: 'Start time for the logs in UTC',
-	required: true,
 });
 
 const endTimeFlag = Flags.string({
 	description: 'End time for the logs in UTC',
-	required: true,
+});
+
+const pastFlag = Flags.string({
+	description: 'Past time window in mins',
+});
+
+const fromFlag = Flags.string({
+	description:
+		'From time in DD-MM-YYYY:HH:MM:SS format to fetch logs from the past. Used as the starting time for the past time duration.',
```

Comment:
```suggestion
		'The from time in DD-MM-YYYY:HH:MM:SS format. It is used to fetch logs from the past and is the starting time for the past time duration.',
```

Reply:
@jhadobe can you suggest how we can inform the customer that this time input  would be inferred based on system timezone.

Reply:
```suggestion
		'The from time in DD-MM-YYYY:HH:MM:SS format based on your system's time zone. It is used to fetch logs from the past and is the starting time for the past time duration.',
```
@dani97 i would suggest this if you mean the user's system timezone.